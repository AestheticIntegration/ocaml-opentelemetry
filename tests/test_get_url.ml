let url = "http://localhost:3000"

let ocurl () =
  let config = Opentelemetry_client_ocurl.Config.make ~url () in
  Opentelemetry_client_ocurl.with_setup ~config () @@ fun () ->
  let url = Opentelemetry_client_ocurl.get_url () in
  print_endline @@ Printf.sprintf "ocurl url = %s" url

let cohttp () =
  let config = Opentelemetry_client_cohttp_lwt.Config.make ~url () in
  Opentelemetry_client_cohttp_lwt.with_setup ~config () @@ fun () ->
  let url = Opentelemetry_client_cohttp_lwt.get_url () in
  print_endline @@ Printf.sprintf "cohttp url = %s" url

let () =
  ocurl ();
  cohttp ()
