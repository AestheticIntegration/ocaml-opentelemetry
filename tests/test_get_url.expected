ocurl url = http://localhost:4318
cohttp url = http://localhost:4318
